src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -272,6 +272,8 @@ internal enum WellKnownType
         System_Span_T,
         System_ReadOnlySpan_T,
 
+        System_Runtime_CompilerServices_SkipLocalsInitAttribute,
```

Comment:
We should group this with the other `System_Runtime_CompilerServices` values.

Reply:
Done.